config/config.yaml

```diff
@@ -3,9 +3,9 @@ release: trusty
 python_path: '/usr/bin/python'
 requirements_file: 'agent_packager/tests/resources/requirements.txt'
 # cloudify_agent_version: 3.1
-cloudify_agent_module: https://github.com/nir0s/cloudify-agent/archive/master.tar.gz
+cloudify_agent_module: https://github.com/cloudify-cosmo/cloudify-agent/archive/master.tar.gz
 core_modules:
-    cloudify_plugins_common: https://github.com/cloudify-cosmo/cloudify-plugins-common/archive/3.1.tar.gz
+    cloudify_plugins_common: https://github.com/cloudify-cosmo/cloudify-plugins-common/archive/master.tar.gz
     # cloudify_rest_client: https://github.com/cloudify-cosmo/cloudify-rest-client/archive/3.1.tar.gz
```

Comment:
i think rest_client must now be used, ie. uncommented and changed to master as well
(see CI)